lib/services/location_service.dart

```diff
@@ -27,4 +31,25 @@ class LocationService {
       distanceFilter: 0,
     );
   }
+
+  // set distance between user and games and order from nearest to furthest
+  List<Game> setAndOrderGamesByDistance(
+      List<Game> games, Location userLocation) {
+    // calculate and set distanceFromUser property in each game
+    final gamesWithDistance = games.map<Game>((game) {
+      game.distanceFromUser = distance(
```

Comment:
I might argue that "distance" is polluting the namespace here but I barely care so totally understand if you give 0️⃣ Fs